### ISSUE TITLE
Error while upgrading miniloaders with the upgrade planner

### ISSUE THREAD

**Issue by NoxyNixie** [mspielberg/factorio-miniloader#6]
This happened while trying to upgrade some miniloaders using the upgrade planner from Klonan: https://mods.factorio.com/mods/Klonan/upgrade-planner:
```
Error while running event upgrade-planner::on_player_selected_area (ID 49)
Error while running event miniloader::on_built_entity (ID 6)
LuaItemStack API call when LuaItemStack was invalid.
stack traceback:
	__miniloader__/compat/upgradeplanner.lua:56: in function 'on_built_entity'
	__miniloader__/control.lua:111: in function <__miniloader__/control.lua:105>
stack traceback:
	__upgrade-planner__/control.lua:805: in function 'player_upgrade'
	__upgrade-planner__/control.lua:582: in function 'on_selected_area'
	__upgrade-planner__/control.lua:559: in function <__upgrade-planner__/control.lua:558
```

**Comment by mspielberg** [comment]
Thanks for the report! This should be fixed in v1.5.8.

**Comment by Fuyukai** [comment]
I'm getting the same issue with a different traceback:

```
123278.027 Error MainLoop.cpp:1019: Exception at tick 23093153: Error while running event upgrade-planner::on_player_selected_area (ID 49)
Entity is not inserter.
stack traceback:
	__upgrade-planner__/control.lua:702: in function 'player_upgrade'
	__upgrade-planner__/control.lua:582: in function 'on_selected_area'
	__upgrade-planner__/control.lua:559: in function <__upgrade-planner__/control.lua:558>
123278.029 Error ServerMultiplayerManager.cpp:95: MultiplayerManager failed: "Error while running event upgrade-planner::on_player_selected_area (ID 49)
Entity is not inserter.
stack traceback:
	__upgrade-planner__/control.lua:702: in function 'player_upgrade'
	__upgrade-planner__/control.lua:582: in function 'on_selected_area'
	__upgrade-planner__/control.lua:559: in function <__upgrade-planner__/control.lua:558>"
```
